### PR TITLE
Release staging v1.4.6

### PR DESCRIPTION
**Pull request by martinstibbe**
## Description

Release staging v1.4.6

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the Terraform contribution guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

**Review by Zuhairahmed — APPROVED** [review]
Where existing INTMDB tickets exist we should include in title of commit to make easier to check that all tickets from JIRA tagged for this release are included. 

For example below are missing tags: 
1. [Add support for use_org_and_group_names_in_export_prefix parameter](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/846/commits/4781cac5be752a2b800cc0f29f9954e367685969) - should be tagged to [INTMDB-385](https://jira.mongodb.org/browse/INTMDB-385)
2. [Update tests related to alert configuratiopn](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/846/commits/aab78b49f9fa424fef1406495548c901cd892f18) - should be tagged to [INTMDB-372](https://jira.mongodb.org/browse/INTMDB-372). also minor typo in commit title "configuratiopn"

Otherwise LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,40 @@
...
+- INTMDB-363 - [Updated Feature] Add serverless backup to mongodbatlas_serverless_instance
+- INTMDB-379 - [Terraform] Release 1.4.6 (both pre and then GA)	
+- INTMDB-381 - [Terraform] Customer is unable to disable backup auto export
+- INTMDB-383 - [Terraform] Update the BYOK/encryption at rest resource
+- INTMDB-385 - use_org_and_group_names_in_export_prefix is not working for a customer
+- INTMDB-386 - [Terraform] Add new role types to invitation verification	
+- INTMDB-371 - [Terraform] Timeout when creating privatelink_endpoint	
+- INTMDB-372 - [Terraform] Fix failing test for testAccMongoDBAtlasAlertConfigurationConfigWithMatchers	
+- INTMDB-358 - [terraform] Upgrade to go1.18
+- INTMDB-391 - [Terraform] Doc Fix for teams.html.markdown
```
Comment (themantissa) [inline CHANGELOG.md]: We need to make sure these are pointing to either an issue or a PR (and the issue should have the PR it links to).  Not going to block on this but please fix before the release.

**Review by themantissa — APPROVED** [review]
LGTM except we need links per my comments.  Please do that before release.

File: CHANGELOG.md (hunk 1 above)
Comment (martinstibbe) [reply CHANGELOG.md]: Fixed changelog

**Review by Zuhairahmed — APPROVED** [review]

**Review by themantissa — APPROVED** [review]
LGTM! Thank you for the Changelog attention.

**Pull request merged**